be/src/exec/vectorized/hdfs_scanner_text.cpp

```diff
@@ -188,11 +188,18 @@ Status HdfsTextScanner::parse_csv(int chunk_size, ChunkPtr* chunk) {
 
         bool has_error = false;
         int num_materialize_columns = _scanner_params.materialize_slots.size();
+        if (_scanner_params.hive_column_names->size() > fields.size()) {
```

Comment:
The size of `hive_column_names` may be larger than `fields` when new columns are added.
The default value should be filled when querying the extra columns that do not exist in the text file.